Sources/SwiftBuildSupport/PIFBuilder.swift

```diff
@@ -130,9 +202,51 @@ public final class PIFBuilder {
 
     private var cachedPIF: PIF.TopLevelObject?
 
+    private func buildPluginTools(
```

Comment:
nit: I find this name a little confusing since this function is not actually performing the build of the tools, might be worth a comment